components/tabs/Tab.js

```diff
@@ -40,6 +42,18 @@ class Tab extends Component {
     }
   };
 
+  renderIcon = () => {
+    if (typeof this.props.icon === 'undefined') {
+      return null;
+    } else if (typeof this.props.icon === 'string') {
+      return (
```

Comment:
Is this div really needed? Also I think handling the string is done inside the FontIcon component so maybe it's not needed here. Did you tried??